src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It looks like now adding a jit specific config value requires adding at two places: src/inc/clrconfigvalues.h and src/jit/jitconfigvalues.h

It would have been great if there is a single jitconfigvalues.h (under src/inc) which in turn is included by clrconfigvalues.h and also used by Jitconfig.h/.cpp